envs/example/ci-full-rhel/group_vars/ceph_monitors.yml

```diff
@@ -1,12 +0,0 @@
----
```

Comment:
Seems we are using rhosp pool instead of ceph pool for ceph monitor nodes, so it doesn't need this configuration.